digit-ui-esbuild/products/dashboard/src/styles/input.css

```
@@ -616,6 +616,226 @@
+    @apply tw-min-w-0 tw-flex-1 tw-overflow-hidden tw-text-ellipsis tw-whitespace-nowrap;
+  }
+
+  .dashboard-menu-item-trailing {
+    @apply tw-inline-flex tw-shrink-0 tw-items-center tw-text-muted-foreground;
+  }
+
+  /* currentColor, not --primary: selection reads as weight + check in the
+     row's own neutral foreground — the filter bar has no brand accent and
+     the menus don't introduce one. */
+  .dashboard-menu-item-check {
+    color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)